tensornetwork/backend/numpy/decompositions.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tensor Decomposition Numpy Implementation."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import numpy as np
+from typing import Optional, Tuple
+
+
+def svd_decomposition(tensor: np.ndarray,
```

Comment:
Can you remove the numpy import, add a argument `np` and pass `self.np` into this function? 

Normally I wouldn't make you do this, but by doing it that way we can get JAX support for free.

Reply:
Oops, yes, I forgot to do this in decompositions. Just make sure typing is okay.